CMakeLists.txt

```diff
@@ -615,4 +627,29 @@ else ()
   endif ()
 endif (IS_64BIT_BUILD EQUAL 1)
 
+
+# The prebuilt IDL headers aren't suitable for 32 bit builds, attempt to use the ones 
+# generated when building corguids by including them before the prebuilt ones
+if (WIN32 AND CLR_CMAKE_PLATFORM_ARCH_I386)
```

Comment:
We should just use the x86 version of the prebuilt header file. I have change in flight to get it updated. Once it is in, this should not be necessary.